pkg/combustion/custom.go

```diff
@@ -72,12 +73,18 @@ func copyCustomFiles(fromDir, toDir string) ([]string, error) {
 		copyMe := filepath.Join(fromDir, entry.Name())
 		copyTo := filepath.Join(toDir, entry.Name())
 
-		info, err := entry.Info()
-		if err != nil {
-			return nil, fmt.Errorf("reading file info: %w", err)
+		var mode os.FileMode
+		if filePermissions == nil {
+			info, infoErr := entry.Info()
+			if infoErr != nil {
+				return nil, fmt.Errorf("reading file info: %w", err)
```

Comment:
This is using the wrong error:
```suggestion
				return nil, fmt.Errorf("reading file info: %w", infoErr)
```